pages/learn/enterprise-integration-patterns/enterprise-integration-patterns/.gitignore

```diff
@@ -19,6 +19,9 @@
 *.tar.gz
 *.rar
 
+.DS_Store
+node_modules/
+
```

Comment:
Remove the .gitignore file.

Reply:
Can remove if needed. But this only effects the files within this directory

Reply:
Deleted it just in case